entities/dashboard/widget.json

```diff
@@ -0,0 +1,128 @@
+        },
+        "feature_flags": {
+            "type": "string",
+            "description": "The feature flags list required to be able to use this widget, seperated by comma"
+        },
+        "module": {
+            "type": "string",
+            "description": "The module that owns the widget"
+        },
+        "component_data": {
+            "type": "object",
+            "description": "a configuration object that is passed to the widget to initialize it. Can vary based on the widget class"
+        },
+        "dimensions": {
+            "min_width": {
+                "type": "integer",
+                "description": "the minimum width of the widget in grid units",
+                "enum": [
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6
+                ]
+            },
+            "min_height": {
+                "type": "integer",
+                "description": "the minimum height of the widget in grid units",
+                "enum": [
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6
+                ]
+            },
+            "height": {
+                "type": "integer",
+                "description": "the default height of the widget"
+            },
+            "width": {
+                "type": "integer",
+                "description": "the default width of the widget"
+            },
+            "max_width": {
+                "type": "integer",
+                "description": "the maximum width of the widget in grid units",
+                "enum": [
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6
+                ]
+            },
+            "max_height": {
+                "type": "int",
+                "description": "the maximum height of the widget in grid units",
+                "enum": [
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6
+                ]
+            }
+        },
+        "created_at": {
+            "type": "string",
+            "description": "The time the call was created"
+        },
+        "updated_at": {
+            "type": "string",
+            "description": "The time the call was last updated"
+        }
+    },
+    "description": "A widget that can be added to the dashboard to display information or functionality",
+    "example": {
+        "uid": "d290f1ee-6c54-4b01-90e6-d701748f0851",
+        "app_code_name": "myappcode",
+        "display_name": {
+            "en": "Payments",
+            "he": "תשלומים"
+        },
+        "permissions": "payments_edit,payments_read",
```

Comment:
these are not names of permissions in the system. it should follow the new permission structure:
clients.clients_list.export
See above my comment about comma seperated